package.json

```diff
@@ -7,6 +7,12 @@
     "test": "echo \"Error: no test specified\" && exit 1",
     "dev": "better-npm-run dev",
     "start": "npm run build && better-npm-run prod",
+    "create-model": "npx sequelize-cli model:generate --name user --attributes name:string,username:string,email:string,password:string",
```

Comment:
Remove the create_model command because it will create the user model only once.